packages/ng-table-builder/src/services/template-parser/template-parser.service.ts

```diff
@@ -149,7 +161,7 @@ export class TemplateParserService<T> {
             stickyLeft: getValidHtmlBooleanAttribute(column.stickyLeft),
             stickyRight: getValidHtmlBooleanAttribute(column.stickyRight),
             customColumn: isCustomKey,
-            width: getValidPredicate(column.width, this.columnOptions?.width),
+            width: fallbackIfEmpty(Number(column.width ?? this.columnOptions?.width ?? undefined), null),
```

Comment:
we really need `undefined`?

Reply:
stupil, i've changed to-number signature, now it receives nullable :0